pages/blog/release-notes-2.3.0.md

```diff
@@ -0,0 +1,116 @@
+channels:
+  some/events:
+    $ref: '#/components/channels/myChannel'
+components:
+  servers:
+    myserver:
+      url: "http://localhost:5000/ws"
+      protocol: ws
+  channels:
+    myChannel:
+      description: "mychannel"
+```
+
+These are added to the many other aspects of the AsyncAPI specification which can be declared as reusable components. You can see the full list in the [Components Object section of the AsyncAPI specification](https://www.asyncapi.com/docs/specifications/v2.3.0#componentsObject).
+
+This new feature was contributed by [Sergio Moya](https://www.linkedin.com/in/smoya). For more detail, see this [pull request](https://github.com/asyncapi/spec/pull/665) and the [Github issue where this change was discussed](https://github.com/asyncapi/spec/issues/660).
+
+
+## Channels are now optional
+
+Related to the above change, and also in support of enabling greater flexibility in how AsyncAPI documents are structured and reused, `channels` are no longer a required section.
+
+This change makes it easier, for example, to have an AsyncAPI document that only contains reusable component definitions.
+
+For example, you could now have a document like this, which you use solely to hold reusable components for use in other documents.
+
+```yaml
+asyncapi: 2.3.0
+info:
+  description: Dictionary for our definitions
+servers:
+  production:
+    $ref: '#/components/servers/myserver'
+components:
+  servers:
+    myserver:
+      url: "http://localhost:5000/ws"
+      protocol: ws
+  channels:
+    myChannel:
+      description: "mychannel"
+    myOtherChannel:
+      description: "another channel"
+```
+
+This could be then used in other documents, like:
+```yaml
+asyncapi: 2.3.0
+info:
+  description: My awesome service API
+servers:
+  myserver:
+    $ref: 'dictionary.yaml#/components/servers/myserver'
+channels:
+  mychannel1:
+    $ref: 'dictionary.yaml#/components/channels/myChannel'
+  mychannel2:
+    $ref: 'dictionary.yaml#/components/channels/myOtherChannel'
+```
+
+For more detail, see the [Github issue where this change was discussed](https://github.com/asyncapi/spec/issues/661).
+
+
+## New protocol bindings
+
+The specification is now extended to support another custom protocol through the bindings feature:
+
+Solace, thanks to [Michael Davis](https://github.com/damaru-inc).
+For more details, check out this [pull request](https://github.com/asyncapi/spec/pull/666) and [binding definition](https://github.com/asyncapi/bindings/tree/master/solace).
+
+
+## Other enhancements
+
+Regular expressions are now Unicode-compliant. This enhancement to the specification from [Sergio Moya](https://www.linkedin.com/in/smoya) means that regular expressions (found in `pattern` and `patternProperties` fields) are now Unicode-compliant (according to ECMA-262).
+
+This improves compatibility with some JSON Schema parsers. For more details, check out the [pull request](https://github.com/asyncapi/spec-json-schemas/pull/145).
+
+
```

Comment:
In light of https://github.com/asyncapi/spec/pull/696 I'm suggesting adding a following deprecation section:

```
## Deprecations

We're deprecating `Channel Item Object` `$ref` field starting from 2.3 version. The plan is to remove the `$ref` field 
from `Channel Item Object` in 3.x major version and allow `Reference Object` to be used wherever the `Channel Item Object` is defined.
```